Dockerfile

```diff
@@ -56,7 +56,11 @@ RUN echo "#!/usr/bin/with-contenv bash" > /etc/cont-init.d/bootstrap_container &
     echo "sed -i '/^R_LIBS_USER=/c\\R_LIBS_USER=/home/'\${USER}'/.rpckg' /usr/local/lib/R/etc/Renviron" >> /etc/cont-init.d/bootstrap_container && \
     echo "sed -i 's#HOST=#HOST='\${EMR_HOST_NAME}'#g' /etc/odbc.ini" >> /etc/cont-init.d/bootstrap_container && \
     echo "sed -i 's#REPLACEME#'\${LIVY_URL}'#g' /etc/skel/.spark_config.yml" >> /etc/cont-init.d/bootstrap_container && \
-    chmod +x /etc/cont-init.d/bootstrap_container && \
+    echo "#!/usr/bin/bash" > /etc/profile.d/rstudio.sh && \
+    echo "echo \"export HADOOP_HOME=\"/opt/dataworks/hadoop-3.3.1\"\" >> /etc/profile.d/rstudio.sh" >> /etc/cont-init.d/bootstrap_container && \
```

Comment:
Minor thing. Some of these lines don't need putting into bootstrap_container. If they don't contain any variables then they can be just put only into /etc/profile.d.  However it should still work fine.

Reply:
I see your point. I will update line 60 and 61 just like 62 in the next update.